web/src/components/confirm-form.jsx

```diff
@@ -71,14 +72,22 @@ class ConfirmForm extends React.Component {
         )}
 
         <section id="confirm-buttons">
+
+          { pendingSentences || readyCount === 0 ? 
+            <SpinnerButton></SpinnerButton> :
+            <button type="submit" disabled={pendingSentences || readyCount === 0}>Confirm</button> 
```

Comment:
since it will show the `SpinnerButton` if `pendingSentences` is true, we can remove the `pendingSentences` check from the disabled check

Reply:
Agree